rbc/tests/heavydb/test_daytimeinterval.py

```diff
@@ -132,7 +133,7 @@ def rbc_generate_series(mgr, start, stop, step, series_output):
 @pytest.mark.parametrize("start, stop, step, order", inputs)
 def test_generate_time_series(heavydb, start, stop, step, order):
 
-    if heavydb.version[:2] >= (6, 2):
+    if heavydb.version[:2] <= (6, 2):
```

Comment:
```suggestion
    if heavydb.version[:2] < (6, 2):
```
or update the skip message (`6.2` -> `6.3`).